README.md

```diff
@@ -173,15 +183,11 @@ Assume this is the XML
    </eos:AdditionalAttributes>
 </gmi:MI_Metadata>
 ```
-##### with this config
+##### with this collection config
```

Comment:
These aren't collection configs though, right? For Cumulus collection configs, the additionalAttributes would be in meta?

Reply:
yes it is; i've added in an image for clarification
https://github.com/podaac/cumulus-metadata-aggregator/blob/207b1aeb236795039e0a14c848aa2209ea6c5ccf/documentation_image/additionalAttributeImage.png